Sources/Components/SaveSearchView/SearchListEmptyViewModel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+public class SearchListEmptyViewModel {
```

Comment:
It might be a good idea to use a `protocol` instead of `class` here.

Reply:
I would probably use `struct` here. With a `protocol` we'll still have to create a struct in the app, which is a bit unnecessary in this case.